var/spack/repos/builtin/packages/vizglow/package.py

```diff
@@ -39,20 +39,73 @@ class Vizglow(Package):
 
     homepage = "http://esgeetech.com/products/vizglow-plasma-modeling/"
 
-    version('2.2a-15', 'be2b5044f30f2b2c3bbe87a0037bf228', expand=False,
+    version('2.2alpha20', '2bef890c66f3a44aaf96f7c96788c89e', expand=False,
+            url="file://{0}/VizGlow_v2.2alpha20-Linux-x86_64-R09December2016-Install".format(os.getcwd()))
+    version('2.2alpha17', '1de268564363e0ee86f9ffff1c3b82e1', expand=False,
+            url="file://{0}/VizGlow_v2.2alpha17-R21November2016-Linux-x86_64-Install".format(os.getcwd()))
+    version('2.2alpha15', 'be2b5044f30f2b2c3bbe87a0037bf228', expand=False,
             url="file://{0}/VizGlow_v2.2alpha15-Linux-x86_64-R31October2016-Install".format(os.getcwd()))
 
+    # depends_on('mesa')  # TODO: mesa build doesn't work for me
+    depends_on('zlib')
+    depends_on('freetype')
+    depends_on('fontconfig')
+    depends_on('libxrender')
+    depends_on('xterm')
+    # Can't get mozjs to build, packagekit -> polkit -> mozjs
+    # depends_on('packagekit+gtk')
```

Comment:
I have WIP packages for packagekit, polkit, and mozjs. Wasn't sure whether or not they were worth adding.